backend/controllers/addUserCaseload.ts

```diff
@@ -28,6 +30,10 @@ const selectCaseloadsFactory = (getUserAssignableCaseloads: any, saveCaseloads:
   }
 
   const post = async (req: Request, res: Response) => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
Can avoid the need for these by adding the userDetails definition to https://github.com/ministryofjustice/hmpps-manage-users/blob/main/backend/%40types/express/index.d.ts#L7

even if it's just typed as a `Record<string, unknown>`